module/Database/src/Service/Member.php

```diff
@@ -182,6 +182,43 @@ public function sendMemberSubscriptionEmail(ProspectiveMemberModel $member): voi
         $this->getMailTransport()->send($message);
     }
 
+    /**
+     * Send an email about the approval to the new member and the secretary
+     */
+    public function sendMemberConfirmedEmail(MemberModel $member): void
+    {
+        $config = $this->config;
+        $config = $config['email'];
+
+        $renderer = $this->getRenderer();
+        $model = new ViewModel([
+            'member' => $member,
+        ]);
+        $model->setTemplate('database/email/member-welcome');
+        $body = $renderer->render($model);
+
+        $html = new MimePart($body);
+        $html->type = "text/html";
+
+        $mimeMessage = new MimeMessage();
+        $mimeMessage->addPart($html);
+
+        $message = new Message();
+        $message->setBody($mimeMessage);
+        $message->setFrom($config['from']);
+        $message->addTo($config['to']['subscription']);
+        $message->setSubject('Membership confirmed: ' . $member->getFullName());
+        $this->getMailTransport()->send($message);
```

Comment:
My thought was that the secretary could create a mail rule to let all emails from gewisdb@gewis.nl whose subject starts with "Membership confirmed" go into a special folder to keep track of what is sent to members. I thought about adding secretary in the BCC of the first message, but then considered it would probably be inconvenient having no reference to the member in the subject. 
I'm fine with both

Reply:
Then we just go for your first option.